index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xml:lang="en" lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>Portfolio Project</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+
+  <div class="header">
+    <img class="logo" src="My Logo.png">
+    <nav class="flex-nav">
+      <a href="#" class="toggleNav"><i class="fa fa-hamburguer"></i>= Menu</a>
+      <ul class="open">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+  </div>
+
+  <div class="wrapper">  
+    <section class="introduction">
+      <h1>I’m Alexandra<br>Glad to see you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <br>
+      <h6>LET’S CONNECT</h6>
+    </section>
+
+    <section class="info">
+      <ul>
+        <li class="media">
+          <a href="http://twitter.com"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li class="media">
+          <a href="http://linkedin.com"><i class="fa fa-linkedin"></i></a>
+        </li>
+        <li class="media">
+          <a href="https://medium.com/"><i class="fa fa-medium"></i></a>
+        </li>
+        <li class="media">
+          <a href="https://github.com/"><i class="fa fa-github"></i></a>
+        </li>
+        <li class="media">
+          <a href="https://www.angellist.com/"><i class="fa fa-angellist"></i></a>
+        </li>
+      </ul>
+    </section>  
+  </div>
```

Comment:
- [ ] For better code organizing I suggest having one **section tag as the parent tag**, and the div element is better used for grouping structures.